python/ray/train/examples/pipeline_parallelism_example.py

```diff
@@ -0,0 +1,352 @@
+import argparse
```

Comment:
Should we also add this to here https://docs.ray.io/en/latest/train/examples.html?

Reply:
Definitely, I'm talking to @maxpumperla about converting this to a notebook before merging. With that, I will address the suggestions made in your main comment about including the README & yaml content.